src/lib.rs

```diff
@@ -182,61 +149,124 @@ impl<C: Config> Plugin for GgrsPlugin<C> {
             ..default()
         });
 
-        app.init_resource::<RollbackTypeRegistry>()
-            .init_resource::<RollbackSnapshots>()
-            .init_resource::<RollbackFrameCount>()
+        app.init_resource::<RollbackFrameCount>()
+            .init_resource::<ConfirmedFrameCount>()
+            .init_resource::<MaxPredictionWindow>()
+            .init_resource::<RollbackOrdered>()
             .init_resource::<LocalPlayers>()
             .init_resource::<FixedTimestepData>()
             .add_schedule(GgrsSchedule, schedule)
             .add_schedule(ReadInputs, Schedule::new())
             .add_systems(PreUpdate, schedule_systems::run_ggrs_schedules::<C>)
-            .add_systems(LoadWorld, load_world)
-            .add_systems(SaveWorld, save_world);
+            .add_plugins((
+                GgrsSnapshotSetPlugin,
+                GgrsChecksumPlugin,
+                GgrsResourceSnapshotCopyPlugin::<Checksum>::default(),
+                GgrsEntitySnapshotPlugin,
+                GgrsComponentSnapshotReflectPlugin::<Parent>::default(),
+                GgrsComponentMapEntitiesPlugin::<Parent>::default(),
+                GgrsComponentSnapshotReflectPlugin::<Children>::default(),
+                GgrsComponentMapEntitiesPlugin::<Children>::default(),
```

Comment:
The names for these plugins are very verbose und hard to read. Is the user ever going to touch them? If so, they probably need a nicer name. Even if not, it still should be a consideration.

Reply:
I agree, they're way too verbose. I've completed the set of extension methods on `App` to allow an end user to not use these plugins by name at all.

Reply:
I think it could be sensible to just drop the Ggrs prefix. Such a shame that Components and Resources need different handling and thus different plugins, otherwise, we could drop that part of the name, too :)

Reply:
Just a thought, but maybe it would be possible to make an API like this:

```rust
app.rollback_components_with_clone::<(Transform, Sprite, Velocity)>();
```

Reply:
I believe that is possible. We'd likely need a `impl_tuple` macro to avoid a extreme amount of repetition, but Bevy has several examples of exactly that macro anyway.